src/main/com/fulcrologic/fulcro/algorithms/merge.cljc

```diff
@@ -87,14 +87,14 @@
       (-> data meta :fulcro/leaf boolean))))
 
 (defn union->query
-  "Turn a union query into a query that attempts to encompass all possible things that might be queried"
+  "Turn a union query into a query that attempts to encompass all possible things that might be queried."
   [union-query]
   (->> union-query vals flatten set vec))
 
 (defn mark-missing
-  "Recursively walk the query and response marking anything that was *asked for* in the query but is *not* in the response as missing.
+  "Recursively walk the query and response marking anything that was *asked for* in the query but is *not* in the response, as missing.
   The merge process (which happens later in the plumbing) looks for these markers as indicators to remove any existing
-  data in the database (which has provably disappeared).
+  data in the database (which has probably disappeared).
```

Comment:
https://www.google.com/search?q=provably&oq=provably&aqs=chrome..69i57j0l5.3392j0j7&sourceid=chrome&ie=UTF-8

Not probably

Reply:
Ahh, I see. Didn't relate it to logic solver kind of usage - will keep this in mind.